### DISCUSSION TITLE
Default view with two languages by translations

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15458] [Archive]
Hi,

is it possible to set a default view by item translations, not with only one language, but with two? (see screenshot)?
![translations_default_view](https://user-images.githubusercontent.com/54798086/188921272-f4087673-1190-4235-bd00-696912becce7.PNG)

Best regards,
Tony

**Comment by rynskiy** [comment]
Is anyone here? :)

**Comment by br41nslug** [comment]
This is currently not possible
